src/commands.mjs

```diff
@@ -194,6 +194,13 @@ export const commands = [
     "key":       "branch --track x x",
     "tags":      "Branching and Merging",
   },
+  {
+    "left":      "workspace",
```

Comment:
My initial reaction was that this doesn't do anything to the local workspace, but only effects the local_repo. I'm curious why it's specified like that?

Reply:
I completely agree...

Perhaps it would be better to have it:

`left = right = workspace`, because your `clone` has `right = workspace`.

That might be a better choice?